MailCore/Utils/NotificationsHelper.swift

```diff
@@ -306,6 +306,20 @@ public enum NotificationsHelper {
     }
 
     private static func getCleanBodyFrom(message: Message) async -> String {
+        if let emojiReaction = message.emojiReaction {
+            let name: String
+            if let senderName = message.from.first?.name, !senderName.isEmpty {
```

Comment:
You can add a guard to get the `message.from.first`.
if this value is empty, you can return nil